src/types.js

```diff
@@ -10,6 +10,67 @@ export type TXOBJ = {
     prevScriptPubKey: string,
   }[],
   outs: { script: string, satoshis: number }[],
+  joinsplit?: {}[], // Does ZencashJS support shielded txn?
+  sc_params?: {
+    vsc_ccout: {
+      epoch_length: number,
+      satoshis: number,
+      address: string,
+      customData: string,
+      wCertVk: string,
+      vFieldElementCertificateFieldConfig: number[],
+      vBitVectorCertificateFieldConfig: number[][],
+      forwardTransferScFee: number,
+      mainchainBackwardTransferScFee: number,
+      mbtrRequestDataLength: number
+    }[],
+    vft_ccout: {
+      scid: string,
+      satoshis: number,
+      address: string
```

Comment:
There is new field called **mcReturnAddress** that should be added also in the bitcore-lib-zen (try to run zen-cli help createrawtransaction)